### PR TITLE
ref(cmd/duffle): ref bundle remove + add test

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

File: cmd/duffle/bundle_remove.go [hunk 1]
```diff
@@ -18,80 +18,93 @@ const bundleRemoveDesc = `Remove a bundle from the local storage.
...
 		Long:    bundleRemoveDesc,
 		Args:    cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			bname := args[0]
+			remove.bundleRef = args[0]
+			remove.home = home.Home(homePath())
+			return remove.run()
+		},
+	}
+	cmd.Flags().StringVarP(&remove.versions, "version", "r", "", "A version or SemVer2 version range")
```
Comment (radu-matei) [inline cmd/duffle/bundle_remove.go]: I know it's from the previous iteration of this command, but it's a really counter-intuitive short version of the flag.

Comment (michelleN) [reply cmd/duffle/bundle_remove.go]: That's good feedback. I'll remove it.

File: cmd/duffle/bundle_remove.go [hunk 2]
```diff
@@ -18,80 +18,93 @@ const bundleRemoveDesc = `Remove a bundle from the local storage.
...
-				fmt.Fprintf(w, "Bundle %q not found. Nothing deleted.", bname)
-				return nil
 			}
-			if err := index.WriteFile(h.Repositories(), 0644); err != nil {
-				return err
-			}
-
-			deleteBundleVersions(vers, index, h, w)
+		}
+		// We can skip writing the file if there is nothing to delete.
```
Comment (radu-matei) [inline cmd/duffle/bundle_remove.go]: _skip writing the file_?

File: cmd/duffle/bundle_remove.go [hunk 3]
```diff
@@ -18,80 +18,93 @@ const bundleRemoveDesc = `Remove a bundle from the local storage.
...
```
Comment (radu-matei) [inline cmd/duffle/bundle_remove.go]: _skip writing [the index] file_?
It took me a while to understand what file it references.

Comment (michelleN) [reply cmd/duffle/bundle_remove.go]: yea agreed. I deleted the comment. I think it's obvious from the code that what is happening is if you don't have anything to delete, then don't touch anything and return

**Pull request merged**